### PR TITLE
Update CI scripts

### PR DESCRIPTION
**Pull request by gigony**
Update CI scripts to use rapids environment.

This is a following up PR for #48.

Closes https://github.com/rapidsai/cucim/pull/45
Closes https://github.com/rapidsai/cucim/pull/46

### CODE REVIEW

**Comment by quasiben** [comment]
Thanks @gigony !

**Review by jakirkham — APPROVED** [review]

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -48,14 +49,19 @@ nvidia-smi
...
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
 
-conda install -c conda-forge conda-build -y
+gpuci_logger "Install dependencies"
+gpuci_conda_retry install -y -c rapidsai-nightly \
+    sysroot_linux-64=2.17 \
+    "cudatoolkit=${CUDA_VER}.*" \
+    "python=${PYTHON_VER}.*" \
```
Comment (ajschmidt8) [inline ci/gpu/build.sh:58]: Is there any reason why `python` is included here? our CI images should have the correct versions of `python` installed by default.

Comment (gigony) [reply ci/gpu/build.sh:58]: Thanks @ajschmidt8! 

Not a good reason.
I think it can be removed.

Comment (jakirkham) [reply ci/gpu/build.sh:58]: Do we include `cudatoolkit` as well? If so, maybe we can drop that too?

Curious if we can drop `sysroot_linux-64` (was trying that as a workaround in my older PR). Maybe we don't need that either

Comment (ajschmidt8) [reply ci/gpu/build.sh:58]: That's a good question. I will check if `cudatoolkit` is already installed in our images. cudf [seems to include it](https://github.com/rapidsai/cudf/blob/branch-21.06/ci/gpu/build.sh#L77-L83). I will verify regardless. Not sure about `sysroot_linux-64`. I'm not familiar with that package.

**Review by ajschmidt8 — APPROVED** [review]

**Comment by quasiben** [comment]
@ajschmidt8 I think we need you to merge in.  I see the failed check:
>  Base branch is not under active development

**Comment by quasiben** [comment]
Hmm,  looks like I was commenting as you were merging.

**Comment by jakirkham** [comment]
Thanks AJ! 😄

**Pull request merged**